cmd/device_info_linux.go

```diff
@@ -38,3 +39,29 @@ func getMac(iface string) (string, error) {
 	}
 	return strings.TrimSpace(string(macForIface)), nil
 }
+
+func getOSVersion() (string, error) {
+	osInfoPath := "/etc/os-release"
+
+	osFile, err := os.ReadFile(osInfoPath)
+	if err != nil {
+		return "", fmt.Errorf("cannot read file %w", err)
```

Comment:
colon before error for consistency:

```suggestion
		return "", fmt.Errorf("cannot read file: %w", err)
```